randovania/games/fusion/logic_database/Sector 1 SRX.txt

```diff
@@ -1244,12 +1249,14 @@ Extra - room_id: [35]
   > Door to Checkpoint Intersection
       Any of the following:
           # Deal with the Pirates
-          Charge Beam or Missiles ≥ 10 or Combat (Intermediate) or Can Freeze Enemies or Can Use Power Bombs
+          Charge Beam or Missiles ≥ 10 or Can Kill Super Geron or Can Use Power Bombs
           All of the following:
               Screw Attack
               # Screw Attack to defeat the Pirates can also (accidentally) destroy the platforms, unless cancelling spins
               Space Jump or Movement (Beginner)
-          # Shinespark from Atmospheric Stabilizer Room
+          # Navigate the room Damageless: https://youtu.be/Lh9D7Ye3rw8
+          Combat (Intermediate)
+          # Shinespark from Atmospheric Stabilizer Room: https://youtu.be/_PmmHN9YvKw
           Speed Booster and Disabled Door Lock Rando and Disabled Entrance Rando
```

Comment:
while you're there doing changes: this should have l0 and the stabilizer event added.

Reply:
by checking the DB where they're used. Altho ideally the long names are changed in a later PR too.
Central - 5
northeast - 2
northwest - 1
southeast - 4
southwest - 3

Reply:
That seems a bit counter-intuitive to need to check the DB to know which is which. That being said, I probably could have noticed that if I poked around a bit more. Thanks for the info!